apps/box/src/screens/Settings/About.screen.tsx

```diff
@@ -0,0 +1,16 @@
+import { FxBox, FxText } from '@functionland/component-library';
+import React from 'react';
+import { SubHeaderText } from '../../components/Text';
+
+export const AboutScreen = () => {
+  return (
+    <FxBox marginHorizontal="20">
```

Comment:
I'm guessing we'll want these to be `FxSafeArea`

Reply:
Yeah, I was going to work on safe area for the full app in the PR after settings.

Reply:
Was thinking of just wrapping screens with a layout component to handle this.